modelon/impact/client/entities/experiment.py

```diff
@@ -413,19 +413,23 @@ def _validate_and_fetch_trajectories(
                 "Please specify the list of result keys for the trajectories of "
                 "intrest!"
             )
-        _assert_experiment_is_complete(self.run_info.status, "Simulation")
         assert_variable_in_result(variables, self.get_variables())
 
         return self._sal.experiment.trajectories_get(
             self._workspace_id, self._exp_id, variables, only_last_point, format.value
         )
 
-    def get_trajectories(self, variables: List[str]) -> Dict[str, Any]:
+    def get_trajectories(
+        self, variables: List[str], verify_simulation_is_complete: bool = True
```

Comment:
`verify_simulation_is_complete` is kind of long name. How about something shorter:
- verify_complete
- assert_complete
- check_simulation_done
- ensure_simulation_complete
- check_finished

Reply:
I will go for ensure_simulation_complete :)

Reply:
Or wait ensure sounds like we wait until simulation is complete.. "check_simulation_done" probably is the more appropriate